server/grpc_service.go

```diff
@@ -1203,8 +1203,9 @@ func (s *GrpcServer) ScatterRegion(ctx context.Context, request *pdpb.ScatterReg
 			FinishedPercentage: uint64(percentage),
 		}, nil
 	}
-
-	region := rc.GetRegion(request.GetRegion().GetId())
+	// TODO: Deprecate it use `request.GetRegionsID`.
+	//nolint
+	region := rc.GetRegion(request.GetRegionId())
 	if region == nil {
 		if request.GetRegion() == nil {
 			return nil, errors.Errorf("region %d not found", request.GetRegion().GetId())
```

Comment:
This should also be changed back? ref: https://github.com/tikv/pd/pull/4582/files#diff-eb404b45d0a569b05f58f4e4bf2b42b32150be75ec2ed72f86330c4b5d739eabL1194